[2023-11-27 14:33:30,620] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: final_project_data_covid_etl.create_fact_district_daily manual__2023-11-27T14:23:20.785587+00:00 [queued]>
[2023-11-27 14:33:30,632] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: final_project_data_covid_etl.create_fact_district_daily manual__2023-11-27T14:23:20.785587+00:00 [queued]>
[2023-11-27 14:33:30,632] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-11-27 14:33:30,632] {taskinstance.py:1357} INFO - Starting attempt 2 of 3
[2023-11-27 14:33:30,632] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-11-27 14:33:30,648] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_fact_district_daily> on 2023-11-27 14:23:20.785587+00:00
[2023-11-27 14:33:30,653] {standard_task_runner.py:52} INFO - Started process 1164 to run task
[2023-11-27 14:33:30,657] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'final_project_data_covid_etl', 'create_fact_district_daily', 'manual__2023-11-27T14:23:20.785587+00:00', '--job-id', '469', '--raw', '--subdir', 'DAGS_FOLDER/get_data.py', '--cfg-path', '/tmp/tmp94gg6nl8', '--error-file', '/tmp/tmphja6xn86']
[2023-11-27 14:33:30,658] {standard_task_runner.py:80} INFO - Job 469: Subtask create_fact_district_daily
[2023-11-27 14:33:30,734] {task_command.py:369} INFO - Running <TaskInstance: final_project_data_covid_etl.create_fact_district_daily manual__2023-11-27T14:23:20.785587+00:00 [running]> on host 9bd194bc0dbc
[2023-11-27 14:33:30,816] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ricky
AIRFLOW_CTX_DAG_ID=final_project_data_covid_etl
AIRFLOW_CTX_TASK_ID=create_fact_district_daily
AIRFLOW_CTX_EXECUTION_DATE=2023-11-27T14:23:20.785587+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-27T14:23:20.785587+00:00
[2023-11-27 14:33:30,856] {logging_mixin.py:115} INFO -    CLOSECONTACT  CONFIRMATION  ...  suspect_meninggal     tanggal
0           274             0  ...                  0  2020-08-05
1           534             0  ...                  0  2020-08-05
2          2127             0  ...                  0  2020-08-05
3             0             0  ...                  0  2020-08-05
4          1295             0  ...                  0  2020-08-05

[5 rows x 20 columns]
[2023-11-27 14:33:30,857] {python.py:173} INFO - Done. Returned value was: None
[2023-11-27 14:33:30,871] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=final_project_data_covid_etl, task_id=create_fact_district_daily, execution_date=20231127T142320, start_date=20231127T143330, end_date=20231127T143330
[2023-11-27 14:33:30,908] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-27 14:33:30,967] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
